### PR TITLE
Script: Update max length for standard opreturn to 80

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

**Comment by eordano** [comment]
Closes #1079

**Comment by braydonf** [comment]
LGTM

File: lib/script/script.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ var Script = function Script(from) {
   }
 };
 
+Script.OP_RETURN_STANDARD_SIZE = 80;
```
Comment (braydonf) [inline lib/script/script.js]: Should this go near the other constants that are defined on Script?

Comment (eordano) [inline lib/script/script.js]: Updated

**Pull request merged**